back/engines/free/email_campaigns/app/views/email_campaigns/idea_published_mailer/campaign_mail.mjml

```diff
@@ -10,9 +10,9 @@
       <!-- Finished action: idea published -->
       <tr>
         <td style="width:22px; padding-bottom:15px">
-          <img style="vertical-align: middle;"
+          <img style="vertical-align: middle; width: 22px; height: 22px;"
```

Comment:
The `td` tag above already has `width:22px`. Are you sure it's needed here too? Maybe `height` would be enough?

Reply:
Maybe. Will try.

Reply:
You are correct. Updated accordingly, except for Twitter icon, which requires a custom ratio.